### ISSUE TITLE
Re-engineering Request Spatial Region Use

### ISSUE THREAD

**Issue by johnbeve** [biomedontology/idobru#6]
It seems _geospatial region_ has been imported from IDO (I assume an older version) and in IDOBRU is a subclass of _one-dimensional spatial region_ with subclass, e.g. _infectious disease free geospatial region_. 

First thing - at least in its present version - IDO doesn't include _geospatial region_.  
Second, assuming IDO imported the term in the past from the Geographical Entity Ontology, the current version of GEO includes _geospatial region_ as a subclass of _material entity_ rather than _immaterial entity_. If IDOBRU will be using that term, then it might be a good idea to import it from the present GEO.  
Third thing, _spatial region_ according to BFO is essentially a class of mathematical or abstract constructs, akin to frames of reference used by physicists. Examples of _one-dimensional spatial region_ include the edge of a cubed shape portion of space. I'm not sure that's what is needed to capture IDOBRU classes such as _infectious disease free geospatial region_. 
Fourth thing, I'd make more substantial suggestions for re-engineering, but there aren't textual definitions for the relevant IDOBRU classes like _infectious disease free geospatial region_, so I'm not exactly sure what sort of entity is needed. If you can give me a better idea of what's intended by introducing the class, I'll do my best to make more constructive suggestions.

**Comment by yongqunh** [comment]
Yes. It appears that some reconstruction is needed. I think this term (infectious disease free geospatial region) was generated by Asiyah many years ago with some use case in mind.

**Comment by linikujp** [comment]
It is for the political regions that are not recognized as a country. Will check on BFO to see which category we will put in.

**Comment by linikujp** [comment]
05/19/2020 meeting with Shane B. John B, Oliver H, Asiyah L.
Decided to use the candidate http://www.ontobee.org/ontology/GENEPIO?iri=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000415 imported from SIO.

**Comment by linikujp** [comment]
Also check Barry S.'s work 
http://ceur-ws.org/Vol-1304/STIDS2014_T01_DonohueEtAl.pdf
